cmd/hub/cmd/build.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"path"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tinkerbell/actions/pkg/git"
+	"github.com/tinkerbell/actions/pkg/img"
+)
+
+type buildOptions struct {
+	context       string
+	containerRepo string
+	dryRun        bool
+	gitRef        string
+	platforms     string
+}
+
+var buildOpts = &buildOptions{}
+
+var buildCmd = &cobra.Command{
```

Comment:
Cobra supports `Example`, do you mind adding documentation about how to run this command? What to expect and so on? Thanks!